utils/find_missing_copyright_headers.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+err=0
+for file in $(find $1 '(' -name '*.cc' -or -name '*.h' ')' -not -path '*third_party*')
+do
+  read line < "$file"
+  if ! [ "${line:0:1}" = "/" ]
```

Comment:
I think you should look for the text "Copyright" in the second line, to make it more specific.